source/jesp.adoc

```diff
@@ -1,21 +1,22 @@
 = Jakarta EE Specification Process
 
-version 1.0. Effective April 9/2019
+version 1.2. Effective July 16/2019
 
 The Jakarta EE Specification Process (JESP) is concerned with the Specification Process as it applies to Specification Projects operating under the purview of the Jakarta EE Working Group. 
 
-The Jakarta EE Specification Committee hereby adopts the https://www.eclipse.org/projects/efsp[Eclipse Foundation Specification Process v1.1] (EFSP) as the Jakarta EE Specification Process with the following modifications:
+The Jakarta EE Specification Committee hereby adopts the https://www.eclipse.org/projects/efsp?version=1.2[Eclipse Foundation Specification Process v1.2] (EFSP) as the Jakarta EE Specification Process with the following modifications:
 
 . Any modification to or revision of this Jakarta EE Specification Process, including the adoption of a new version of the EFSP, must be approved by a Super-majority of the Specification Committee, including a Super-majority of the Strategic Members of the Jakarta EE Working Group, in addition to any other ballot requirements set forth in the EFSP.
 . All specification committee approval ballot periods will have the minimum duration as outlined below
 .. Creation Review: 7 calendar days;
 .. Plan Review:  7 calendar days;
-.. Progress Review: 30 calendar days;
-.. Release Review: 30 calendar days;
-.. Service Release: 30 calendar days; and
+.. Progress Review: 14 calendar days;
+.. Release Review: 14 calendar days;
+.. Service Release Review: 14 calendar days; and
 .. JESP Update: 7 calendar days.
+. A ballot will be declared invalid and concluded immediately in the event that the Specifiation Team withdraws from the corresponding review.
 . Specification Projects must engage in at least one Progress or Release Review  per year while in active development.
 
-All development that modifies content in the javax namespace must occur within the scope of a Specification Project operating under the purview of the Jakarta EE Working Group’s Specification Committee and must implement the process as defined by the most recently adopted revision of the JESP.
+All development that modifies content in the `javax` namespace must be moved to a `jakarta` namespace. All `jakarta` namespace development much occur within the scope of a Specification Project operating under the purview of the Jakarta EE Working Group’s Specification Committee and must implement the process as defined by the most recently adopted revision of the JESP.
```

Comment:
The "namespace development much" phrase needs to be "namespace development must"